src/tests/actions/articleActions.test.js

```diff
@@ -43,6 +49,26 @@ describe('articleActions', () => {
     );
   });
 
+  it('should return a specific article', async () => {
+    const res_data = {
+      article: {
+        author: { bio: 'Nice' },
+        body: 'Plain and simple',
+      },
+    };
+    const slug = 'an-article';
+    mock.onGet(`api/articles/${slug}`)
+      .reply(200, res_data);
+    fetchSpecificArticle(slug)(store.dispatch);
+
+    await flushAllPromises();
```

Comment:
Rather than awaiting this function call, you can await the one on line 62 to complete.

Reply:
@RonKbS the change that you asked me to implement did not work as expected.